common/src/main/webapp/sass/explore.scss

```diff
@@ -991,3 +994,104 @@ table.explore-border-table {
   }
 }
 
+// --------------
+// Timing windows
+// --------------
+@mixin timing-window-list {
+  margin-bottom: 0.5em;
+  border: solid 1px var(--under-tab-border-color);
+  min-height: var(--timing-windows-height);
+  height: var(--timing-windows-height);
+  grid-area: table;
+}
+
+.timing-windows-body {
+  --timing-windows-height: 200px;
+
+  padding: 0.5em;
+  display: grid;
+  grid-template:
+    'table table' min-content
+    'editor add' min-content / 1fr min-content;
+
+  @media (min-width: common.$mobile-responsive-cutoff) {
+    --timing-windows-height: 300px;
+  }
+
+  .timing-windows-header {
+    display: none;
+    height: 0;
+
+    & tr {
+      height: 0;
+    }
+  }
+
+  .timing-windows-empty {
+    @include timing-window-list;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: large;
+  }
+
+  .timing-windows-table {
+    @include timing-window-list;
+  }
+
+  .timing-window-editor {
+    @include react-datepicker-input;
+    @include react-datepicker-wrapper;
+
+    grid-area: editor;
+    padding: 0.5em;
+    font-size: smaller;
+
+    & span {
```

Comment:
Are all of these `&`s needed? The `&` is needed when you want to add something like another class to the outer selector. But here, it seems like these are children.

Reply:
Hmm, I guess I missundrestood, will review